README.md

```diff
@@ -97,105 +89,60 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
```

Comment:
- [ OPTIONAL ] have done well adding all sections of the readme. In my opinion, Some sections are not relevant to this particular project and can be safely deleted while keeping the professionality of your project intact. 🤝  
![built](https://user-images.githubusercontent.com/99656687/228146358-d7326f44-e9b0-404e-a9a5-db7c4eeb7ef5.png)
 - You only have the front-end and it's built with CSS & HTML, I suggest you replace `react.js` with `HTML` & `CSS`
 - You can also delete the `backend` & `database` sections since they are not applicable to your project